### ISSUE TITLE
Unable to deploy Sonarqube 9.4 with chart helm 

### ISSUE THREAD

**Issue by pbq** [mc1arke/sonarqube-community-branch-plugin#609]
**Describe the bug**
I am not able to deploy Sonarqube 9.4 (with sonarsource provided helm chart) with the branch plugin.

**To Reproduce**
1. Edit the values.yaml file with 
```yaml
plugins:
  install: 
    - https://github.com/mc1arke/sonarqube-community-branch-plugin/releases/download/1.11.0/sonarqube-community-branch-plugin-1.11.0.jar

jvmOpts: "-javaagent:./extensions/plugins/sonarqube-community-branch-plugin-1.11.0.jar=web"
jvmCeOpts: "-javaagent:./extensions/plugins/sonarqube-community-branch-plugin-1.11.0.jar=ce"
```
2. Deploy on kubernetes

**Expected behavior**
Sonarqube is starting.

**Screenshots**
![image](https://user-images.githubusercontent.com/4497414/170268330-586ea8ac-76d4-4a03-a130-d999b954bf2d.png)

**Software Versions**
 - SonarQube Version: 9.4 (Sonarqube chart 3.0.0)
 - Plugin Version: 1.11.0

**Additional context**
I'm suspecting Sonarqube to load the jvm options before plugins are moved from `./extensions/download` to `./extensions/plugins`

**Comment by rgembalik** [comment]
Funnily enough, I wanted to create the same issue, but I saw this one, so while preparing my comment I stumbled upon solution.

```yaml
...
plugins:
  install:
  - https://github.com/mc1arke/sonarqube-community-branch-plugin/releases/download/1.11.0/sonarqube-community-branch-plugin-1.11.0.jar
  lib:
  - sonarqube-community-branch-plugin-1.11.0.jar
jvmOpts: "-javaagent:/opt/sonarqube/lib/common/sonarqube-community-branch-plugin-1.11.0.jar=web"
jvmCeOpts: "-javaagent:/opt/sonarqube/lib/common/sonarqube-community-branch-plugin-1.11.0.jar=ce"
...
```
Note the changed paths on `jvmOpts` and `jvmCeOpts`, the absolute path on paths  to jar and additional `lib` entry, which is supposed to copy the plugin to `lib/common`. 

Out of all possible combinations, this one finally started. I hope it will spare headache to everyone else involved. 

@mc1arke Do you think that is something that we can add to the official documentation?

**Comment by pbq** [comment]
> Funnily enough, I wanted to create the same issue, but I saw this one, so while preparing my comment I stumbled upon solution.
> 
> ```yaml
> ...
> plugins:
>   install:
>   - https://github.com/mc1arke/sonarqube-community-branch-plugin/releases/download/1.11.0/sonarqube-community-branch-plugin-1.11.0.jar
>   lib:
>   - sonarqube-community-branch-plugin-1.11.0.jar
> jvmOpts: "-javaagent:/opt/sonarqube/lib/common/sonarqube-community-branch-plugin-1.11.0.jar=web"
> jvmCeOpts: "-javaagent:/opt/sonarqube/lib/common/sonarqube-community-branch-plugin-1.11.0.jar=ce"
> ...
> ```
> 
> Note the changed paths on `jvmOpts` and `jvmCeOpts`, the absolute path on paths to jar and additional `lib` entry, which is supposed to copy the plugin to `lib/common`.
> 
> Out of all possible combinations, this one finally started. I hope it will spare headache to everyone else involved.
> 
> @mc1arke Do you think that is something that we can add to the official documentation?

I will give it a try on monday. Thank you for your answer !

**Comment by pbq** [comment]
> Funnily enough, I wanted to create the same issue, but I saw this one, so while preparing my comment I stumbled upon solution.
> 
> ```yaml
> ...
> plugins:
>   install:
>   - https://github.com/mc1arke/sonarqube-community-branch-plugin/releases/download/1.11.0/sonarqube-community-branch-plugin-1.11.0.jar
>   lib:
>   - sonarqube-community-branch-plugin-1.11.0.jar
> jvmOpts: "-javaagent:/opt/sonarqube/lib/common/sonarqube-community-branch-plugin-1.11.0.jar=web"
> jvmCeOpts: "-javaagent:/opt/sonarqube/lib/common/sonarqube-community-branch-plugin-1.11.0.jar=ce"
> ...
> ```
> 
> Note the changed paths on `jvmOpts` and `jvmCeOpts`, the absolute path on paths to jar and additional `lib` entry, which is supposed to copy the plugin to `lib/common`.
> 
> Out of all possible combinations, this one finally started. I hope it will spare headache to everyone else involved.
> 
> @mc1arke Do you think that is something that we can add to the official documentation?

@rgembalik, your solution is working. Thank you very much !

**Comment by mc1arke** [comment]
> @mc1arke Do you think that is something that we can add to the official documentation?

I'm happy for someone to raise a PR with additional guidance on configuring Helm if they want

**Comment by rgembalik** [comment]
@mc1arke I prepared a "fix" in #626.
